ddtrace/writer.py

```diff
@@ -41,7 +41,7 @@ def write(self, spans=None, services=None):
             self._traces.add(spans)
 
         if services:
-            self._services.add(services)
+            pass
```

Comment:
You can remove the `if services:` part too, just remove it all.

Also, worth making this a no-op as well?
https://github.com/DataDog/dd-trace-py/blob/4e5acbd8ca1d4087291d9d9c1709b61651bba230/ddtrace/api.py#L116

Reply:
Sure. Was truly going for the one-liner, but it'll be small enough just to remove the rest.